### PR TITLE
Gh-408 Update home and intro pages

### LINKED ISSUE #408: Update Home Page and Intro Pages
The v2 docs currently have a fairly basic home page which could do with some improvement by using HTML rather than markdown to create a unique page with buttons for different guide sections.

Each section since the restructure also has an introduction page which is currently blank so needs an overview of what the section contains and who its intended reader is.

### PR DESCRIPTION
**Pull request by tb06904**
Created a new custom home page to standout a bit from the usual docs pages that has simple links into each of the guides.

Added info to the user and admin introduction pages to give quick overview and some links to key topics.

Recommend you clone branch and view the site to see new page.

# Related Issue

- Resolve #408

### CODE REVIEW

File: docs/home_page_override/home.html [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    <div class="ql-button">
+        <a href="development-guide/introduction/">
+            <h2>
+                <svg xmlns="http://www.w3.org/2000/svg" height="28" viewBox="0 -960 960 960" width="28">
+                    <path
+                        d="m384-336 56-57-87-87 87-87-56-57-144 144 144 144Zm192 0 144-144-144-144-56 57 87 87-87 87 56 57ZM200-120q-33 0-56.5-23.5T120-200v-560q0-33 23.5-56.5T200-840h560q33 0 56.5 23.5T840-760v560q0 33-23.5 56.5T760-120H200Zm0-80h560v-560H200v560Zm0-560v560-560Z" />
+                </svg>
+                Developer Guide
+            </h2>
+            <p>Useful information for developers of Gaffer and general project architecture.</p>
```
Comment (GCHQDeveloper314) [inline docs/home_page_override/home.html]: Could put something in here about contributions as the ways of working page is under this section

File: docs/home_page_override/home.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    <div class="ql-button">
+        <a href="administration-guide/introduction/">
+            <h2>
+                <svg xmlns="http://www.w3.org/2000/svg" height="28" viewBox="0 -960 960 960" width="28">
+                    <path
+                        d="M680-280q25 0 42.5-17.5T740-340q0-25-17.5-42.5T680-400q-25 0-42.5 17.5T620-340q0 25 17.5 42.5T680-280Zm0 120q31 0 57-14.5t42-38.5q-22-13-47-20t-52-7q-27 0-52 7t-47 20q16 24 42 38.5t57 14.5ZM480-80q-139-35-229.5-159.5T160-516v-244l320-120 320 120v227q-19-8-39-14.5t-41-9.5v-147l-240-90-240 90v188q0 47 12.5 94t35 89.5Q310-290 342-254t71 60q11 32 29 61t41 52q-1 0-1.5.5t-1.5.5Zm200 0q-83 0-141.5-58.5T480-280q0-83 58.5-141.5T680-480q83 0 141.5 58.5T880-280q0 83-58.5 141.5T680-80ZM480-494Z" />
+                </svg>
+                Administrators Guide
+            </h2>
+            <p>Guides on how to set up and deploy Gaffer instances and apply custom configuration to them.</p>
```
Comment (GCHQDeveloper314) [inline docs/home_page_override/home.html]:
```suggestion
            <p>Guides on how to set up, deploy, and configure Gaffer instances.</p>
```

**Review by GCHQDeveloper314 — COMMENTED** [review]
New homepage looks good, only minor comments

**Review by GCHQDeveloper314 — APPROVED** [review]

File: docs/administration-guide/introduction.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Administration Guide
+
+This guide is intended for more administrators of a Gaffer deployment and more
```
Comment (cn337131) [inline docs/administration-guide/introduction.md]:
```suggestion
This guide is aimed at administrators of a Gaffer deployment and more
```
Phrasing of this first sentence is a little weird and could do with a little rewrite :)

**Review by cn337131 — APPROVED** [review]

**Pull request merged**